docs/ADOPTING_DARIA.md

```diff
@@ -0,0 +1,43 @@
+# Adopting DARIA
+
+If you are a fund considering adopting DARIA as your case management system of record, this may answer common questions about how things work. We also have a sandbox/training app where no actual patient data lives for experimenting. [FILL IN ANY NECESSARY DETAIL]
+
+## The case management workflow
+
+DARIA's case management workflow was designed to mirror existing practices used in case management. Volunteer case managers answer patient calls coming in through a hotline phone number and voicemail. DCAF case management software helps the case managers keep track of patient status and needs.
+
+The workflow is similar to many Customer Relationship Manager (CRM) systems. After patients leave a voicemail, the case managers return their calls and assist patients in navigating the process of scheduling an appointment and raising funds. If the patient is short on money, after a series of phone calls we issue a pledge to support their visit to a clinic. If a patient takes advantage of the pledge and complete their abortion, our accounting team logs that as a fulfillment.
+
+Read more about our workflow at: 
+https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/DCAF_101.md
+
+## Administration toolkit
+
+DARIA provides a number of workflows to administer and report on your funds activities. Analysts are able to export an anonymized spreadsheet for analysis; administrators are able to keep track of which clinics your fund  are and aren't working with; and your accounting team can use a specialized workflow to mark which pledges have been cashed.
+
```

Comment:
The following are customizations that the DARIA app provides out of the box. With the exception of hotlines, these can all be managed by your fund within the app.

* Number of hotlines
* Clinics
* External Pledge Sources (i.e. funds, other organizations, clinic discounts that contribute to pledges)
* Fax service options
* Insurance options
* Language options (i.e. languages your fund can accommodate)
* Pledge limit help text options (e.g. 1st trimester (7-12 weeks): $100, etc.)
* Referred by options
* Links to additional resources pertinent to your organization

Reply:
This is stuff that I've compiled to email to people and I think it's helpful in a fund's assessment of how much DARIA can be tailored to fit their needs. If you think it's too in the weeds, that's also fine. I was erring on the side of more info up front, but if you think this is better as a less is more type of thing, perhaps it'd be better to just point people to check out the admin tools > config management when checking out the sandbox to get a feel for the level of customization. Happy to defer to your judgment here.

Reply:
Yeah, I think less is more at the point they're reading this document. If they're mulling over customization when they're poking around sandbox you're dead on, we can point them toward config panel then, but let's aim for that to rise up organically.